templates/github/.ci/scripts/publish_plugin_pypi.sh.j2

```diff
@@ -7,6 +7,14 @@ cd "$(dirname "$(realpath -e "$0")")"/../..
 
 set -euv
 
+export VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin {{ plugin_app_label }} -r '.versions[] | select(.component == $plugin) | .version')
+export response=$(curl --write-out %{http_code} --silent --output /dev/null https://pypi.org/project/pulp-npm-client/$VERSION/)
```

Comment:
please don't forget to remove hard-coded npm plugin
s/pulp-npm-client/{{ plugin_dash }}-client